sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.2.0 (2020-01)
+- Bumped to service version 2019-07-07.
+- Shortened Diagnostic Scope names.
```

Comment:
I'm not sure this is worth calling out as nobody uses our distributed tracing yet (ditto on the others).

Reply:
Removed.  I didn't know what this was.